src/main/java/org/opensha/commons/data/WeightedValue.java

```diff
@@ -9,4 +11,12 @@ public WeightedValue(E value, double weight) {
 		this.value = value;
 		this.weight = weight;
 	}
-}
\ No newline at end of file
+	
+	@Override
+	public String toString() {
+		if (value instanceof IntensityMeasureRelationship) {
+			return ((IntensityMeasureRelationship) value).getName();
+		}
+		return value.getClass().getName();
+	}
+}
```

Comment:
Oof, I don't like this hack to make `WeightedValue` know about a specific implementation (`IntensityMeasureRelationship`). How about you check if `value implements Named` and then return `((Named)value).getName()`?

Actually probably best to do this hierarchy:

```java
if (value instanceof ShortNamed)
    return ((ShortNamed)value).getShortName();
else if (value instanceof Named)
    return ((Named)value).getName();
else
    return value.toString(); // better than just the class name, as implementation may already give good toStrings
```

This will accomplish what you're after because `IntensityMeaureRelationship` implements `ShortNamed` (which in turn extends `Named`)